blocks/story-feed-author-content-source-block/package.json

```diff
@@ -1,29 +1,27 @@
 {
-  "name": "@wpmedia/story-feed-author-content-source-block",
-  "version": "5.14.1",
-  "description": "Content source block for story feed queries by author",
-  "author": "Joe Grosspietsch <joe.grosspietsch@washpost.com>",
-  "homepage": "https://github.com/WPMedia/arc-themes-blocks",
-  "license": "CC-BY-NC-ND-4.0",
-  "main": "index.js",
-  "files": [
-    "sources"
-  ],
-  "publishConfig": {
-    "registry": "https://npm.pkg.github.com/",
-    "access": "public"
-  },
-  "repository": {
-    "type": "git",
-    "url": "ssh://git@github.com/WPMedia/arc-themes-blocks.git",
-    "directory": "blocks/story-feed-author-content-source-block"
-  },
-  "scripts": {
-    "test": "echo \"Error: run tests from root\" && exit 1",
-    "lint": "eslint --ext js --ext jsx sources"
-  },
-  "gitHead": "83fb8d6685958d85ddbfcfbf01d9a5c864cd6c95",
-  "peerDependencies": {
-    "@wpmedia/resizer-image-block": "*"
-  }
+	"name": "@wpmedia/story-feed-author-content-source-block",
+	"version": "5.14.1",
+	"description": "Content source block for story feed queries by author",
+	"author": "Arc XP - Themes",
+	"homepage": "https://github.com/WPMedia/arc-themes-blocks",
+	"license": "CC-BY-NC-ND-4.0",
+	"main": "index.js",
```

Comment:
We can remove the `index.js` from main as we do not support importing these

Reply:
Will remove